lib/src/screens/home/feedTab.dart

```diff
@@ -12,8 +21,111 @@ class FeedTab extends StatefulWidget {
 }
 
 class _FeedTabState extends State<FeedTab> {
+  static final DateFormat postDateFormatter = DateFormat('h:m a');
+
+  final List<FeedItem> feedItems = [
+    FeedItem(
+      name: 'Lecture 1',
+      owner: 'Calculus I - AAB',
+      body: 'Please do this IMMEDIATELY',
+      type: FeedItemType.Lecture,
+      itemId: '54t3t3',
+      postDate: DateTime.now().add(Duration(hours: 4)),
+      attachmentUrl:
+          'https://images.all-free-download.com/footage_preview/mp4/kitty_68.mp4',
+    ),
+    FeedItem(
+      name: 'Homework 5',
+      owner: 'English 101',
+      body: 'Never do this',
+      type: FeedItemType.Homework,
+      itemId: '54t3t3',
+      postDate: DateTime.now().add(Duration(hours: 2)),
+    ),
+  ];
+
   @override
   Widget build(BuildContext context) {
-    return Text("Base");
+    final size = MediaQuery.of(context).size;
+
+    return Container(
+      width: size.width,
+      height: size.height,
+      child: ListView.builder(
+        padding: EdgeInsets.all(20),
+        itemCount: feedItems.length,
+        itemBuilder: (context, i) => buildItem(context, feedItems[i]),
+      ),
+    );
+  }
+
+  Widget buildItem(BuildContext context, FeedItem item) {
+    VideoPlayerController? videoPlayerController;
+    // if (item.type == FeedItemType.Lecture && item.attachmentUrl != null) {
+    //   videoPlayerController =
+    //       VideoPlayerController.network(item.attachmentUrl!);
+    //   await videoPlayerController.initialize();
+    //}
+
+    return Padding(
+      padding: EdgeInsets.all(20),
+      child: Card(
+        child: InkWell(
+          onTap: () => {},
+          child: Container(
+            padding: EdgeInsets.all(5),
+            child: Column(
+              children: [
+                Row(
+                  children: [
+                    Text('${item.owner} posted ${item.name}'),
+                    Spacer(),
+                    Text('${postDateFormatter.format(item.postDate)}',
+                        style: TextStyle(color: Colors.grey[400])),
+                  ],
+                ),
+                SizedBox(height: 10),
+                Text(
+                  item.body,
+                  textAlign: TextAlign.center,
+                ),
+                SizedBox(height: 10),
+                SizedBox(height: 10),
```

Comment:
```suggestion
                SizedBox(height: 20),
```
Not sure why there are 2....

Reply:
I think I was planning on adding something in between there, but I forgot. It may have been the video player/image container.

Reply:
Merged